src/textual/css/_style_properties.py

```diff
@@ -782,11 +782,13 @@ def __set__(self, obj: StylesBase, color: Color | str | None):
         elif isinstance(color, str):
             try:
                 parsed_color = Color.parse(color)
-            except ColorParseError:
+            except ColorParseError as error:
                 raise StyleValueError(
                     f"Invalid color value '{color}'",
-                    help_text=color_property_help_text(self.name, context="inline"),
-                )
+                    help_text=color_property_help_text(
+                        self.name, context="inline", error=error
+                    ),
+                ) from error
```

Comment:
added this ` from error` while I was there; it might be useful one day to be able to get the parent error simply by reading its `__cause__` attribute?
(to be honest I was mostly trained to that discipline of specifying "cause" errors by years of Pylint usage, with its [raise-missing-from](https://pylint.pycqa.org/en/latest/messages/warning/raise-missing-from.html) rule enabled ^^)

Reply:
I think in this instance, the previous ColorParseError doesn't add anything. The full stack-trace will never be displayed to the user.